liquibase-standard/src/main/java/liquibase/change/core/CreateProcedureChange.java

```diff
@@ -240,10 +246,58 @@ public InputStream openSqlStream() throws IOException {
      */
     @Override
     public CheckSum generateCheckSum(ChecksumVersions version) {
-        return generateCheckSum(this.procedureText, version);
+        if (version == ChecksumVersions.V8) {
+            return generateCheckSumV8(version);
+        }
+        return generateCheckSumV9(this.procedureText, version);
```

Comment:
Maybe generateCheckSumV9 should be called just "generateCheckSum()" or "generateCheckSumLatest()"? As when we release checksum v10 we may not change all those methods, then it will be generating v9 and v10 .